src/shared/components/players/Player.vue

```diff
@@ -84,6 +84,9 @@ watch(remoteVideoState, (state, prevState) => {
 
             break;
         case playerState.PLAYING:
+            if (store.state.video.currentTime) {
+                player.value.seekTo(store.state.video.currentTime);
+            }
```

Comment:
Add empty line after if-block

```suggestion
            }

```